### ISSUE TITLE
Kubernetes Ingress with Cert-Manager Task Broken

### ISSUE THREAD

**Issue by GregHanson** [istio/istio#21792]
**Bug description**

Following steps here: https://preliminary.istio.io/docs/tasks/traffic-management/ingress/ingress-certmgr/

Using beta.4 release:
```
istioctl manifest apply \
  --set values.gateways.istio-ingressgateway.sds.enabled=true \
  --set values.global.k8sIngress.enabled=true \
  --set values.global.k8sIngress.enableHttps=true \
  --set values.global.k8sIngress.gatewayName=ingressgateway
```
Ingress gateway fails to become ready due to failed LDS config from pilot.

logs from ingress gateway:
```
[Envoy (Epoch 0)] [2020-03-03 22:38:33.276][21][warning][config] [external/envoy/source/common/config/grpc_subscription_impl.cc:87] gRPC config for type.googleapis.com/envoy.api.v2.Listener rejected: Error adding/updating listener(s) 0.0.0.0_443: Invalid path: /etc/istio/ingressgateway-certs/tls.crt

```
Gateway with SIMPLE TLS rule is created , but I'm guessing the corresponding cert/secret was never created, see here in [preconfigured.yaml](https://github.com/istio/istio/blob/release-1.5/manifests/gateways/istio-ingress/templates/preconfigured.yaml)
```
{{ if .Values.global.k8sIngress.enableHttps }}
  - port:
      number: 443
      protocol: HTTPS
      name: https-default
    tls:
      mode: SIMPLE
      serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
      privateKey: /etc/istio/ingressgateway-certs/tls.key
    hosts:
    - "*"
{{ end }}
```

[ ] Configuration Infrastructure
[ X ] Docs
[ X ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ X ] Test and Release
[ X ] User Experience
[ ] Developer Infrastructure

**Environment where bug was observed (cloud vendor, OS, etc)**
IKS

**Comment by howardjohn** [comment]
I think we are properly mounting the secret, so it sounds like its not being created by certmanager?

**Comment by howardjohn** [comment]
This code path is not in 1.6 anymore so this would apply to 1.5 only

**Comment by howardjohn** [comment]
This is fixed in 1.6

**Comment by kramvan1** [comment]
@howardjohn   Was this ever fixed in 1.5?  Ref to that change?
